tests/integration/simple_integration_test.go

```diff
@@ -0,0 +1,164 @@
+package integration_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/random"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"io/ioutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"net/http"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+var (
+	namespace = "k8ssandra"
+	releaseName string
```

Comment:
This is a minor nit, but I find the `releaseNameCluster` variable name a bit confusing. Initially I thought it was the name of the C* cluster and/or CassandraDatacenter. And `releaseName` does not indicate which release/chart it corresponds to. I would prefer something more explicit, albeit a bit verbose, e.g., `k8ssandraReleaseName` and `k8ssandraClusterReleaseName`.